### PR TITLE
Example bounding sphere using macroquad

### PR DESCRIPTION
**Pull request by Vrixyz**
- builds on top of https://github.com/dimforge/parry/pull/257

[Screencast from 09-04-2024 10:50:47 AM.webm](https://github.com/user-attachments/assets/44209789-b0e3-4df5-9a71-70ea53278bea)

[Screencast from 09-04-2024 02:55:32 PM.webm](https://github.com/user-attachments/assets/6590b735-8104-4511-ae56-9b42d8fdf33f)

### CODE REVIEW

File: crates/parry3d/examples/bounding_sphere3d.rs [hunk 1]
```diff
@@ -1,36 +1,113 @@
...
+        });
+
+        /*
+         * Initialize the shapes.
+         */
+        let cube1 = Cuboid::new(Vector3::repeat(0.5));
+        let cube2 = Cuboid::new(Vector3::new(0.5, 1.0, 0.5));
+
+        let cube1_pos = na_from_mquad(lissajous_3d(elapsed_time)) * 4f32;
+        let cube1_pos = Isometry3::translation(cube1_pos.x, cube1_pos.y, cube1_pos.z);
```
Comment (sebcrozet) [inline crates/parry3d/examples/bounding_sphere3d.rs:35]: Does this work?
```suggestion
        let cube1_pos = Isometry3::from(cube1_pos);
```

File: crates/parry3d/examples/common_macroquad.rs
Comment (sebcrozet) [inline crates/parry3d/examples/common_macroquad.rs:1]: It would be great to have comments for every function from this module.

**Review by sebcrozet — APPROVED** [review]
Thanks! I mostly just looked at the bounding_sphere files, as I assume the other changes originate from the PRs this depends on.

File: crates/parry3d/examples/common_macroquad.rs
Comment (Vrixyz) [reply crates/parry3d/examples/common_macroquad.rs:1]: I'll open a new PR for that

**Pull request merged**